### ISSUE TITLE
Installing via Visual Studio Nuget and downloading package gives different version of dll?

### ISSUE THREAD

**Issue by mbooh** [NuGet/Home#4863]
Hi! 
I have a strange problem. 
If I download our packages: https://www.nuget.org/packages/Amesto.AutoConnect.Core/ and then look at the dll in lib it is changed today 21th of March.
But when I use Visual Studio Nuget manager the dll in the packages folder get change date 10th of January.
Does anyone have any idea why this is happening? 
Thanks!

/Kristoffer

**Comment by emgarten** [comment]
The older package could be coming from your cache, use ``nuget.exe locals -clear all`` and try the restore again.

**Comment by mbooh** [comment]
I'm not sure if the nuget command would have solved my problems but in this directory:
C:\Users\Kiwi.SIT\.nuget\packages
I found old stuff that had conflicting version number. I'm not sure how the got there but I rearrenged some stuff some weeks into my project and I probably reseted the version in some way.
Any way, when I deleted all that was there it worked just fine. Does the "-clear all" empty this folder as well?
